Controls/Mail/Mailman.c

```diff
@@ -11,8 +13,21 @@
  * Once the letter is deposited, signal the semaphore (effectively putting up the flag).
  * Print "Finished!" at the end.
  */
+//extern char* mailbox;
 void depositLetter(void) {
 	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+	//int i;
+	//char max=random()%255;//different lengths at diff times
+	for(int i=0;i<255;i++){
+		mailbox[i]=64+rand()%63;
```

Comment:
Again, 64 here could be a magic number (also, it's the ascii value of **@**, not **A** which I don't think is intended). Something nice could be to do:
`
	mailbox[i]='A'+rand()%63;
`

What does 63 mean here? rand()%63 will generate a random number from 0 to 62, inclusive. What characters are you trying to include in your random number generation?